Cargo.toml

```diff
@@ -1,106 +1,64 @@
-[package]
-name = "typstyle"
+[workspace.package]
 version = "0.12.10"
+rust-version = "1.80"
```

Comment:
is this msrv? Or are we locking rust version explicitly?

Reply:
idk, I just take that of typst. Omit this field as before? Or use a newer version?